schemas/2016-03-01/Microsoft.Insights.json

```diff
@@ -14,7 +14,7 @@
         "type": {
           "type": "string",
           "enum": [
-            "microsoft.insights/alertrules"
+            "Microsoft.Insights/alertrules"
```

Comment:
>Microsoft.Insights/alertrules [](start = 13, length = 29)

The Microsoft.Insights tests are failing on this change. I guess you'll need to update the tests as well.